oaknational/Oak-Web-Application
# Fix/video tracking

## Description

- fixes implementation which was broken before and prone to stale values for event properties
- adds the remaining properties, we now have:
![image](https://user-images.githubusercontent.com/12934669/188882142-6d18fa17-3142-4cec-93ae-d4f654175bc8.png)


## Issue(s)

Following up #639 

## How to test

1. Go to {cloud link}
2. Play the video, pus it, go to the end etc
3. You should see correct properties getting fired in Avo and posthog!

## Checklist
